crates/apollo-router/Cargo.toml

```
@@ -10,16 +10,17 @@ name = "router"
 path = "src/main.rs"
 
 [features]
-default = ["otlp-tonic"]
-otlp-tonic = [
+default = ["otlp-grpc"]
+# activates OTLP/gRPC
+otlp-grpc = [
```

Not a useful comment
```suggestion
otlp-grpc = [
```